tests/integration/it_utils.py

```diff
@@ -124,6 +145,74 @@ async def sign_and_reliable_submission_async(
     return response
 
 
+def send_timed_reliable_submission(
+    transaction: Transaction, use_json_client: bool = True
+) -> Response:
+    """
+    Allows integration tests for sync clients to utilize reliable submission on
+    standalone rippled node. A timer to close ledger is initialized before
+    running `send_reliable_submission_sync` to ensure the ledger closes and the
+    submission process finishes.
+
+    Arguments:
+        transaction: the signed transaction to submit to the ledger. Requires a
+            `last_ledger_sequence` param.
+        use_json_client: boolean for choosing json or websocket client.
+
+    Returns:
+        The response from the server, as a Response object.
+    """
+    client = _choose_client(use_json_client)
+    SyncTestTimer(LEDGER_ACCEPT_TIME, client.request, (LEDGER_ACCEPT_REQUEST,)).start()
+    return send_reliable_submission_sync(transaction, client)
+
+
+async def send_timed_reliable_submission_async(
+    transaction: Transaction, use_json_client: bool = True
+) -> Response:
+    """
+    Allows integration tests for async clients to utilize reliable submission on
+    standalone rippled node. A timer to close ledger is initialized before
+    running `send_reliable_submission_async` to ensure the ledger closes and the
+    submission process finishes.
+
+    Arguments:
+        transaction: the signed transaction to submit to the ledger. Requires a
+            `last_ledger_sequence` param.
+        use_json_client: boolean for choosing json or websocket client.
+
+    Returns:
+        The response from the server, as a Response object.
+    """
+    client = _choose_client_async(use_json_client)
+    AsyncTestTimer(LEDGER_ACCEPT_TIME, client, LEDGER_ACCEPT_REQUEST)
+    return await send_reliable_submission_async(transaction, client)
+
+
+def accept_ledger(use_json_client: bool = True) -> None:
```

Comment:
I think this would be better if instead of `send_timed_reliable_submission_async`, you used the `accept_ledger` methods in the tests.